swagconnect/bitbucket/connectors.py

```diff
@@ -13,4 +21,60 @@ class BitbucketConnector(CustomOAuth2Adapter):
     scope = settings.SWAGAUTH_SETTINGS[provider_id]['SCOPE']
 
 
+class BitbucketAPIConnector(BaseBitbucket, BaseAPIConnector):
```

Comment:
Do not derieve this class from `BaseBitbucket`. Move the core logic from the `BaseBitbucket` to the here